src/encoded/regulome_search.py

```diff
@@ -607,12 +611,16 @@ def region_search(context, request):
             result['timing'].append({'nearby_snps': (time.time() - begin)})  # DEBUG: timing
             begin = time.time()                                              # DEBUG: timing
             # NOTE: Needs all hits rather than 'released' or set reduced by facet selection
-            evidence = atlas.regulome_evidence(all_hits['datasets'])
-            features = {k: k in evidence
-                        for k in ['ChIP', 'DNase', 'PWM', 'Footprint',
-                                  'eQTL', 'dsQTL', 'PWM_matched',
-                                  'Footprint_matched']}
+            evidence = atlas.regulome_evidence(all_hits['datasets'], chromosome, int(start), int(end))
             regdb_score = atlas.regulome_score(all_hits['datasets'], evidence)
+            features = evidence.copy()
+            features.update({
+                k: k in evidence
+                for k in [
+                    'ChIP', 'DNase', 'PWM', 'Footprint', 'eQTL', 'dsQTL',
+                    'PWM_matched', 'Footprint_matched'
+                ]
+            })
```

Comment:
This whole bit seems a little clunky.    Why are we copying the evidence Dict then adding back 8 properties?  Seems like it should be one or the other.    Also this is in there twice, should be factored out for the two views.

Reply:
`features` is the exact input into the new scoring algorithm. So I removed details and converted values of those binary variables to boolean. Good to have a function to convert `evidence` to `features` @hitz ?